gap_linux.go

```diff
@@ -45,8 +46,8 @@ func (a *Adapter) DefaultAdvertisement() *Advertisement {
 //
 // On Linux with BlueZ, it is not possible to set the advertisement interval.
 func (a *Advertisement) Configure(options AdvertisementOptions) error {
-	if a.properties != nil {
-		panic("todo: configure advertisement a second time")
+	if a.started {
+		panic("cannot configure advertisement that is already started")
```

Comment:
I wonder if this condition should return an error instead of panicking.

Reply:
That seems reasonable. I will make that change.

Reply:
Done. Thanks for the feedback!